sdk/helper/keysutil/consts.go

```diff
@@ -47,6 +48,7 @@ var (
 	}
 
 	HashFuncMap = map[HashType]func() hash.Hash{
+		HashTypeNone:    nil,
```

Comment:
Isn't that guarded by `prehashed=true`, which is why we enforce that?

Reply:
> a bit of a red flag imho.

Not really; if you look at what the test is doing:

```go
...
.t.Run(testName, func(t *testing.T) { test_RSA_PSS(t, p, rsaKey, hashType, marshalingType) })
...
```

because this NoOID is only relevant when we're doing a PKCS#1v1.5 signature... and the test is explicitly doing a PSS signature... this isn't really a red flag, so much as a fundamental limitation of PSS (which doesn't support NoOID type signatures, because of the way the MFG1 mask generation works...).

Reply:
@stevendpclark I've added a test for PKCS#1v1.5 signatures here in keysutil, including for NoOID. Let me know what you think.

Reply:
Yeah better, still not loving it but I can't think of a better way around the problem at this time so 👍